crowdin.yml

```diff
@@ -54,5 +54,17 @@ files: [
    "dest" : "/App/WireMainNavigationUI/Resources/Accessibility.xcstrings",
    "translation" : "%original_path%/%original_file_name%",
    "multilingual": true,
-  },
+ },
+ {
+  "source" : "/WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings",
+  "dest" : "/App/WireFolderPickerUI/Resources/Localizable.xcstrings",
```

Comment:
@caldrian Do you know if I have to manually create these items on crowdin or will they be auto created when the localization action runs?

Reply:
They will be created.